tests/stix_transmission/splunk/test_class.py

```diff
@@ -83,7 +107,33 @@ def test_query_response(self, mock_query_response, mock_api_client):
         assert query_response['success'] is True
         assert 'search_id' in query_response
         assert query_response['search_id'] == "1536672851.4012"
- 
+
+    @patch('stix_shifter.stix_transmission.src.modules.splunk.spl_api_client.APIClient.create_search')
+    def test_query_response_exception(self, mock_query_response, mock_api_client):
+        mock_api_client.return_value = None
+        mocked_return_value = '{"sid":"1536672851.4012"}'
+        mock_query_response.return_value = SplunkMockResponse(201, mocked_return_value)
+        mock_query_response.side_effect = Exception('an error occurred creating query')
```

Comment:
the text is a little bit confusing, the text is not checked in the test but it looks pretty similar to the text in the checking. I would replace it with some dummy text and checked it as well.